IBX-154: Optional alt text requirement for Image FT (#1433)
ezsystems/developer-documentation: docs/tutorials/enterprise_beginner/1_get_a_starter_website.md

![New image Field in the Article Content Type](img/enterprise_tut_image_in_article_ct.png)

!!! note "Alternative text requirement"

>     In some cases you might want to require an alternative text for the image. 

Author: ```suggestion
    It is recommended that you require an alternative text for the image. 
```

    To do this, select the "Alternative text is required" checkbox.

> 

Reviewer: Selecting alternative text as mandatory should be our recommendation here. It's usually (and by default) used as `alt` attribute value and it's its value must not be empty. 

Ref. https://html.spec.whatwg.org/multipage/images.html#general-guidelines
